gcexport.py

```diff
@@ -903,12 +975,12 @@ def main(argv):
                 pass
                 # Display which entry we're skipping.
                 print('Skipping Garmin Connect activity ', end='')
-                print('(' + str(current_index) + '/' + str(total_to_download) + ') ', end='')
+                print('(' + str(current_index) + sep + str(total_to_download) + ') ', end='')
                 print('[' + str(actvty['activityId']) + '] \n', end='')
             else:
                 # Display which entry we're working on.
                 print('Garmin Connect activity ', end='')
-                print('(' + str(current_index) + '/' + str(total_to_download) + ') ', end='')
+                print('(' + str(current_index) + sep + str(total_to_download) + ') ', end='')
```

Comment:
```suggestion
                print(f'({current_index}{sep}{total_to_download}) ', end='')
```

Reply:
Python 2 compatibility update
```suggestion
                print('(', current_index, sep, total_to_download, ') ', sep='', end='')
```

Reply:
Just curious, what is the benefit of comma-separating the arguments to `print()` here?

Reply:
Performance. Strings are immutable objects in Python, so by using concatenation (`+` operator) you actually create new objects under the hood, which requires more CPU operations and memory allocation, this also may invokes more work for garbage collector too. As passing already existed objects to print, you just pass references.

Reply:
Thanks! Done (also for the other places where applicable)